docs/dev-guide/contrib-guides-taskfiles.md

```diff
@@ -0,0 +1,161 @@
+# Taskfiles
+
+Follow the guidelines below when writing and updating [Taskfiles]. Note that neither the guidelines
+nor examples are written as a rigorous specification, but they should be sufficient to understand
+the spirit of each guideline.
+
+## Variable naming
+
+1. Variables should be named using `SCREAMING_SNAKE_CASE`. 
+2. Variables in the global scope (as opposed to a task's scope) should be prefixed with `G_` to
+   avoid conflicts with local variables.
+
+**Example**
+
+```yaml
+vars:
+  G_BUILD_DIR: "build"
+
+binaries:
+  vars:
+    BIN_BUILD_DIR: "{{.G_BUILD_DIR}}/bin"
+```
+
+## Paths in commands
+
+All paths used in commands should be quoted.
+
+**Example**
+
+```yaml
+my-task:
+  cmds:
+    - "touch '{{.FILE_PATH}}'"
```

Comment:
I assume we don't mention "strings should be quoted" in the first place because "have automated linting processes", right?

Reply:
Yep